app/models/like.rb

```diff
@@ -1,4 +1,10 @@
 class Like < ApplicationRecord
   belongs_to :post
   belongs_to :user
```

Comment:
- [ ] As per the given _ERD_, the _Like_ **association** should have _Like_ belong to _User_ **through** _Author_, an alias of _User_, like below:
`belongs_to :author, class_name: 'User'` 

    Kindly ensure to improve upon this aspect.